py/server/deephaven/column.py

```diff
@@ -32,47 +32,122 @@ def __repr__(self):
         return self.name
 
 
-@dataclass
-class Column:
-    """ A Column object represents a column definition in a Deephaven Table. """
-    name: str
-    data_type: DType
-    component_type: DType = None
-    column_type: ColumnType = ColumnType.NORMAL
+class ColumnDefinition(JObjectWrapper):
+    """A Deephaven column definition."""
+
+    j_object_type = _JColumnDefinition
+
+    @staticmethod
+    def of(
```

Comment:
1. `of` is idiomatic Java.  `from_<stuff>` is idiomatic python. (e.g. https://dev.to/p0intman/alternative-constructors-in-python-58gb or https://realpython.com/python-multiple-constructors/)
2. Missing pydoc.